doc/v2.md

```diff
@@ -0,0 +1,20 @@
+## Guide for Migration from v1 to v2
+
+`v2` contains lots of refactoral changes to improve the performance and configuration by introducing [`@logdna/logger`](https://www.npmjs.com/package/@logdna/logger), [`@logdna/env-config`](https://www.npmjs.com/package/@logdna/env-config), and [`eslint-config-logdna`](https://www.npmjs.com/package/eslint-config-logdna). There are several changes in naming and handling environment variables as well. All environment variables are documented [here](./env.md)
+
+### Renamed Variables
+* `LOGDNA_KEY` has been replaced with [`INGESTION_KEY`](./env.md#ingestion_key)
+* `LOGDNA_HOSTNAME` has been replaced with [`HOSTNAME`](./env.md#hostname)
+* `LOGDNA_TAGS` has been replaced with [`TAGS`](./env.md#tags)
+* `LOGDNA_URL` has been split into multiple new variables: [`INGESTION_ENDPOINT`](./env.md#ingestion_endpoint), [`INGESTION_HOST`](./env.md#ingestion_host), [`INGESTION_PORT`](./env.md#ingestion_port), and [`SSL`](./env.md#ssl).
+* `LOGDNA_BATCH_INTERVAL` has been replaced with [`FLUSH_INTERVAL`](./env.md#flush_interval)
+* `LOGDNA_BATCH_LIMIT` has been replaced with [`FLUSH_LIMIT`](./env.md#flush_limit)
+
+### Removed Variables
+* `LOGDNA_FREE_SOCKET_TIMEOUT`, `LOGDNA_MAX_REQUEST_RETRIES`, `LOGDNA_REQUEST_RETRY_INTERVAL`, and `LOGDNA_MAX_REQUEST_TIMEOUT` have been removed since the connection gets handled within the logger client
```

Comment:
Replace "have been removed" with "were removed" and replace "since" with "because."

Also replace "gets handled" with "is handled."

Add period (.) to end of sentence.

Reply:
Done!